compiler/rustc_data_structures/src/sync.rs

```diff
@@ -360,6 +413,129 @@ cfg_if! {
     }
 }
 
+pub fn for_each_in<T: IntoIterator>(t: T, mut for_each: impl FnMut(T::Item)) {
```

Comment:
Can `par_for_each_in` function inside the `cfg_if` be a `use` of this one? It's basically a copy-paste, isn't it?

Reply:
We make branches inside `par_for_each_in` in the #[cfg(parallel_compiler)] now so don't need `for_each_in` anymore